lib/ui/fixtures/ui_test.dart

```diff
@@ -373,10 +373,10 @@ void convertPaintToDlPaint() {
 void _convertPaintToDlPaint(Paint paint) native 'ConvertPaintToDlPaint';
 
 @pragma('vm:entry-point')
-void hooksTests() {
-  void test(String name, VoidCallback testFunction) {
+void hooksTests() async {
+  Future<void> test(String name, FutureOr<void> testFunction()) async {
```

Comment:
Wow, I didn't know Dart could do this, is this idiomatic as opposed to having `FutureOr<void> Function() testFunction`?

Reply:
Looks like the `Function` syntax is now preferred
(see https://dart.dev/guides/language/effective-dart/design#prefer-using-function-type-syntax-for-parameters)